tower/src/builder/mod.rs

```diff
@@ -688,6 +688,121 @@ impl<L> ServiceBuilder<L> {
     {
         self
     }
+
+    /// Convert the resulting service into a [`Service`] + [`Send`] trait object.
+    ///
+    /// This applies the [`BoxService:layer()`] layer. See that method for more details.
```

Comment:
nit: wrong link
```suggestion
    /// This applies the [`BoxService::layer()`] layer. See that method for more details.
```

Reply:
also, TIOLI, but most of the other `ServiceBuilder` methods use a slightly different phrasing "This wraps ...":
https://github.com/tower-rs/tower/blob/ef9066b567af0191007dc94c2c04573ead5fbadd/tower/src/builder/mod.rs#L193-L194
so, maybe:
```suggestion
    /// This wraps the inner service with the [`Layer`] returned by [`BoxService::layer()`]. 
    /// See that method for more details.
```